PythonTool/TagExpansion.py

```diff
@@ -191,12 +223,27 @@ def expand_vector_tags(mesh_file, main_dir_name = None):
     mb_exp = core.Core()
     mb_exp.load_file(mesh_file)
 
-    # Retrieve the lists of scalar and vector tags on the mesh.
-    try:
-        hexes_exp, scal_tags_exp, vec_tags_exp = get_tag_lists(mb_exp, types.MBHEX)
-    except LookupError as e:
-        print(str(e))
-        exit()
+    # Retrieve the lists of scalar and vector tags on the expansion mesh.
+    if element_type is None:
+        # Search for hex elements in the mesh.
+        element_type = "hex"
+        try:
+            hexes_exp, scal_tags_exp, vec_tags_exp = get_tag_lists(mb_exp, element_type, types.MBHEX)
+        except LookupError as e:
+            print(str(e))
+            exit()
+    else:
+        # Ensure the user entered a valid MB element type.
+        type_int = elements.get(element_type.lower())
```

Comment:
I do see that checking for the user-supplied element type in the PyMOAB `.types` module could be a little tricky here, so it makes sense why you defined the `elements` data structure. Let's talk more about this in person at some point. I think we can figure something relatively clean out to use the PyMOAB types.